src/plots/Mesh/MeshAttributes.code

```diff
@@ -1,4 +1,3 @@
-Target: xml2atts
 Function: ChangesRequireRecalculation
```

Comment:
I used xmledit to make these changes, surprised to see the Target dropped?

Reply:
xml2atts is the default target, so I'm not surprised.  If I ever hand-edit a .code file I usually add in the Target especially if I am adding code for other targets, just to be consistent.
Might be a good idea to modify xmledit to always supply the target.